### PR TITLE
Self-Service Partners Training Issues fix

### PR DESCRIPTION
**Pull request by meggmumbi**
1. Add the CurrentVL variable to AggregateOptimizeCurrentRegimens and AggregateOptimizeStartRegimens
2. Created a new aggregate of TPT from FactTPT -: load_aggregate_hts_tbscreening_TPT
3. Added  -CountCPIMSUniqueIdentifier to AggregateOVCCount
4. Added TX_New variable to the Aggregate HTS table:- AggregateHTSUptake

### CODE REVIEW

**Review by nobert-mumo — CHANGES_REQUESTED** [review]
@meggmumbi Majorly LGTM! Please rem to add the new columns introduced to the data dictionary.

**Review by nobert-mumo — APPROVED** [review]

**Review by cbrianbet — APPROVED** [review]

File: Scripts/REPORTING/load_aggregate_hts_uptake.sql
Comment (Marymary-dev) [inline Scripts/REPORTING/load_aggregate_hts_uptake.sql:1]: Just a question why are we joining on DimARTOutcome?

**Review by Marymary-dev — COMMENTED** [review]
-I was able to follow the code until the sub queries section 
-Why are we joining on DimARTOutcome

**Review by Marymary-dev — APPROVED** [review]
-I was able to follow the code until the sub queries section 
-Why are we joining on DimARTOutcome

File: Scripts/REPORTING/load_aggregate_hts_uptake.sql
Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_hts_uptake.sql:1]: actually, good question I think this is redundant here @meggmumbi

**Pull request merged**